<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PostProcessingSettingsWidget</class>
 <widget class="QWidget" name="PostProcessingSettingsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>683</width>
    <height>514</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0,0">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QCheckBox" name="enablePostProcessing">
       <property name="text">
        <string>Enable Post Processing</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="reload">
       <property name="text">
        <string>&amp;Reload Shaders</string>
       </property>
       <property name="icon">
        <iconset resource="resources/resources.qrc">
         <normaloff>:/icons/view-refresh.png</normaloff>:/icons/view-refresh.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="loadPreset">
       <property name="text">
        <string>Load Preset</string>
       </property>
       <property name="icon">
        <iconset resource="resources/resources.qrc">
         <normaloff>:/icons/document-save.png</normaloff>:/icons/document-save.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="savePreset">
       <property name="text">
        <string>Save Preset</string>
       </property>
       <property name="icon">
        <iconset resource="resources/resources.qrc">
         <normaloff>:/icons/document-open.png</normaloff>:/icons/document-open.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Post Processing Chain</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="PostProcessingChainConfigWidget" name="widget" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>681</width>
        <height>390</height>
       </rect>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PostProcessingChainConfigWidget</class>
   <extends>QWidget</extends>
   <header>duckstation-qt/postprocessingchainconfigwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
